src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
I analysed this issue and found a bigger missconception. See CELDEV-516
Please add a migration according to CELDEV-516 to migrate all XWikiPreferences with activated webSearch classCollection to have after executing the migration a correctly activated websearch ClassPackage in there XWikiPreferences.